src/eos-dropbox-app

```diff
@@ -41,32 +41,61 @@ def launch_dropbox_daemon():
     subprocess.Popen([DROPBOX_LAUNCHER])
 
 
+def get_default_dropbox_directory():
+    default_dir = os.path.expanduser(DROPBOX_DEFAULT_DIR)
+
+    if os.path.isdir(default_dir):
+        return default_dir
+
+    # If no Dropbox folder found, that means that the user has not
+    # configured Dropbox yet, so there's not much we can do now
+    return None
+
+
 def get_dropbox_directory():
-    dropbox_dir = os.path.expanduser(DROPBOX_DEFAULT_DIR)
+    dropbox_dir = None
+    config_path = os.path.expanduser(DROPBOX_CONFIG)
 
-    logging.info("Looking for the configured Dropbox directory...")
-    with open(os.path.expanduser(DROPBOX_CONFIG), 'r') as config:
-        data = config.read()
+    logging.info("Looking for Dropbox configuration...")
+    if not os.path.exists(config_path):
+        logging.warning('Could not find Dropbox configuration')
```

Comment:
Maybe this should just be debug info as it sounds like a serious warning, although it will be followed by _User has not configured Dropbox yet_